src/data/finsupp/basic.lean

```diff
@@ -1634,6 +1670,33 @@ lemma map_domain_map_range [add_comm_monoid N] (f : α → β) (v : α →₀ M)
 let g' : M →+ N := { to_fun := g, map_zero' := h0, map_add' := hadd} in
 add_monoid_hom.congr_fun (map_domain.add_monoid_hom_comp_map_range f g') v
 
+lemma sum_update_add {α β : Type*} [add_comm_monoid α] [add_comm_monoid β]
+  (f : ι →₀ α) (i : ι) (a : α) (g : ι → α → β) (hg : ∀ i, g i 0 = 0)
+  (hgg : ∀ (j : ι) (b₁ b₂ : α), g j (b₁ + b₂) = g j b₁ + g j b₂) :
```

Comment:
Total nitpick but when I see `b₁ b₂` I have to fight with myself not to see them as terms of `β`
```suggestion
  (hgg : ∀ (j : ι) (a₁ a₂ : α), g j (a₁ + a₂) = g j a₁ + g j a₂) :
```